<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quasi</class>
 <widget class="QWidget" name="Quasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>880</width>
        <height>580</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoLoad" stdset="0">
            <bool>true</bool>
           </property>
           <property name="workspaceSuffixes" stdset="0">
            <stringlist>
             <string>_res</string>
             <string>_red</string>
             <string>_sqw</string>
            </stringlist>
           </property>
           <property name="fileBrowserSuffixes" stdset="0">
            <stringlist>
             <string>_res.nxs</string>
             <string>_red.nxs</string>
             <string>_sqw.nxs</string>
            </stringlist>
           </property>
           <property name="showLoad" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblProgram">
           <property name="text">
            <string>Program</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbProgram">
           <item>
            <property name="text">
             <string>Lorentzians</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stretched Exponential</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="workspaceSuffixes" stdset="0">
            <stringlist>
             <string>_red</string>
             <string>_sqw</string>
            </stringlist>
           </property>
           <property name="fileBrowserSuffixes" stdset="0">
            <stringlist>
             <string>_red.nxs</string>
             <string>_sqw.nxs</string>
            </stringlist>
           </property>
           <property name="showLoad" stdset="0">
            <bool>false</bool>
           </property>
           <property name="autoLoad" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblSample">
           <property name="text">
            <string>Sample:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblResolution">
           <property name="text">
            <string>Resolution:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="fitOptions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Fit Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QCheckBox" name="chkFixWidth">
              <property name="text">
               <string>Fix Width</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="chkUseResNorm">
              <property name="text">
               <string>Use ResNorm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="MantidQt::API::FileFinderWidget" name="mwFixWidthDat" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="label" stdset="0">
               <string/>
              </property>
              <property name="fileExtensions" stdset="0">
               <stringlist>
                <string>.dat</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="cbBackground">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>Sloping</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Flat</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Zero</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="chkElasticPeak">
                <property name="text">
                 <string>Elastic Peak</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkSequentialFit">
                <property name="text">
                 <string>Sequential Fit</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResNorm" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoLoad" stdset="0">
               <bool>true</bool>
              </property>
              <property name="workspaceSuffixes" stdset="0">
               <stringlist>
                <string>_ResNorm</string>
               </stringlist>
              </property>
              <property name="fileBrowserSuffixes" stdset="0">
               <stringlist>
                <string>_ResNorm.nxs</string>
               </stringlist>
              </property>
              <property name="showLoad" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblBackground">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Background:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="treeSpace"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="plotPane">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="showLegend" stdset="0">
              <bool>true</bool>
             </property>
             <property name="canvasColour" stdset="0">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lblPreviewSpec">
                <property name="text">
                 <string>Preview Spectrum: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spPreviewSpectrum">
                <property name="maximum">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="legendSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbPlotPreview">
                <property name="text">
                 <string>Plot Current Preview</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MantidQt::CustomInterfaces::RunView" name="runWidget" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_44">
            <item>
             <widget class="QGroupBox" name="gbOutput">
              <property name="title">
               <string>Output Options</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lblPlotResult">
                 <property name="text">
                  <string>Plot Result: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbPlot">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>All</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Amplitude</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>FWHM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Prob</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbPlot">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Plot</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_18">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbSave">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Save Result</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::API::FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/FileFinderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::RunView</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Spectroscopy/RunWidget/RunView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
